stix_shifter_modules/tanium/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1058 @@
+        "object": "alert"
+    },
+    "groupingId":
+    {
+        "key": "x-ibm-finding.x_grouping_id",
+        "object": "alert"
+    },
+    "intelDocRevisionId":
+    {
+        "key": "x-ibm-finding.x_intel_doc_revision_id",
+        "object": "alert"
+    },
+    "scanConfigId":
+    {
+        "key": "x-ibm-finding.x_scan_config_id",
+        "object": "alert"
+    },
+    "scanConfigRevisionId":
+    {
+        "key": "x-ibm-finding.x_scan_config_revision_id",
+        "object": "alert"
+    },
+    "path":
+    {
+        "key": "x-ibm-finding.x_path",
+        "object": "alert"
+    },
+    "receivedAt":
+    {
+        "key": "x-ibm-finding.x_received_at",
+        "object": "alert"
+    },
+    "suppressedAt":
+    {
+        "key": "x-ibm-finding.x_suppressed_at",
+        "object": "alert"
+    },
+    "findingId":
+    {
+        "key": "x-ibm-finding.x_finding_id",
+        "object": "alert"
+    },
+    "ackedAt":
+    {
+        "key": "x-ibm-finding.x_acked_at",
+        "object": "alert"
+    },
+    "firstEIDResolutionAttempt":
+    {
+        "key": "x-ibm-finding.x_first_eid_resolution_attempt",
+        "object": "alert"
+    },
+    "lastEIDResolutionAttempt":
+    {
+        "key": "x-ibm-finding.x_last_eid_resolution_attempt",
+        "object": "alert"
+    },
+    "createdAt": [
+        {
+            "key": "x-ibm-finding.time_observed",
+            "object": "alert"
+        },
+        {
+            "key": "first_observed"
+        }
+    ],
+    "updatedAt":
+    {
+        "key": "last_observed"
+    },
+    "computerIpAddress": [
+        {
+            "key": "ipv4-addr.value",
+            "object": "target-ip",
+            "transformer": "CheckIPv4"
+        },
+        {
+            "key": "ipv6-addr.value",
+            "object": "target-ip",
+            "transformer": "CheckIPv6"
+        },
+        {
+            "key": "x-ibm-finding.dst_ip_ref",
+            "object": "alert",
+            "references": "target-ip"
+        },
+        {
+            "key": "x-oca-event.ip_refs",
+            "object": "event",
+            "references": [
+                "ip"
+            ]
+        },
+        {
+            "key": "x-oca-asset.ip_refs",
+            "object": "target-asset",
+            "references": [
+                "target-ip"
+            ]
+        }
+    ],
+    "computerName": [
+        {
+            "key": "x-oca-asset.hostname",
+            "object": "target-asset"
+        },
+        {
+            "key": "x-oca-event.host_ref",
+            "object": "event",
+            "references": "target-asset"
+        }
+    ],
+    "state": {
+        "key": "x-oca-event.outcome",
+        "object": "event"
+    },
+    "severity": [
+        {
+            "key": "x-ibm-finding.severity",
+            "object": "alert",
+            "transformer": "ConvertTextSeverityToNumberValue"
+        },
+        {
+            "key": "x-oca-event.severity",
```

Comment:
kinda redundant to create "x-oca-event.severity" since you already mapped to "x-ibm-finding.severity"